api/src/main/resources/cb_search_api.yaml

```diff
@@ -110,6 +110,47 @@ paths:
           schema:
             $ref: "#/definitions/CriteriaListResponse"
 
+  /v1/cohortbuilder/{cdrVersionId}/criteria/drug/{name}:
+    parameters:
+      - $ref: '#/parameters/cdrVersionId'
+    get:
+      tags:
+        - cohortBuilder
+      description: Returns criteria tree with the specified name
+      operationId: "getDrugBrandOrIngrediantByName"
```

Comment:
nit: change 'Ingrediant' to 'Ingredient' to avoid possible future issues

Reply:
Nice catch. Done.